gui/default/index.html

```diff
@@ -504,9 +505,12 @@ <h4 class="panel-title">
                 <button type="button" class="btn btn-sm btn-danger pull-left" ng-click="override(folder.id)" ng-if="folderStatus(folder) == 'outofsync' && folder.type == 'sendonly'">
                   <span class="fas fa-arrow-circle-up"></span>&nbsp;<span translate>Override Changes</span>
                 </button>
-                <button type="button" class="btn btn-sm btn-danger pull-left" ng-click="revert(folder.id)" ng-if="canRevert(folder.id)">
+                <button type="button" class="btn btn-sm btn-danger pull-left" ng-click="revert(folder.id)" ng-if="folder.type === 'receiveonly' && hasLocalChanged(folder)">
                   <span class="fa fa-arrow-circle-down"></span>&nbsp;<span translate>Revert Local Changes</span>
                 </button>
+                <button type="button" class="btn btn-sm btn-danger pull-left" ng-click="revertEncModal(folder.id)" ng-if="folder.type === 'receiveencrypted' && hasLocalChanged(folder)">
+                  <span class="fa fa-arrow-lock"></span>&nbsp;<span translate>Revert Not Encrypted Items</span>
```

Comment:
This is not really a revert, this would be a delete presumably, which is destructive?

We have revert local changes, which undoes changes but I suspect it does not delete extra files does it?

Reply:
Yes, revert local changes does delete locally added files as well as reverting files to the global state. I'll rename it to delete - specific is always better.